external-scripts.json

```diff
@@ -1,11 +1,7 @@
 [
   "hubot-diagnostics",
-  "hubot-google-translate",
   "hubot-hackbot",
   "hubot-heroku-keepalive",
-  "hubot-maps",
-  "hubot-pugme",
   "hubot-redis-brain",
-  "hubot-rules",
-  "hubot-shipit"
+  "hubot-rules"
```

Comment:
Is [`hubot-rules`](https://github.com/hubot-scripts/hubot-rules) a serious script?